pydantic_ai_slim/pydantic_ai/builtin_tools.py

```diff
@@ -394,9 +412,19 @@ class MCPServerTool(AbstractBuiltinTool):
     def unique_id(self) -> str:
         return ':'.join([self.kind, self.id])
 
+    @property
+    def label(self) -> str:
+        return f'MCP: {self.id}'
+
 
 def _tool_discriminator(tool_data: dict[str, Any] | AbstractBuiltinTool) -> str:
     if isinstance(tool_data, dict):
         return tool_data.get('kind', AbstractBuiltinTool.kind)
     else:
         return tool_data.kind
+
+
+# used in a couple places
+def get_builtin_tool_types() -> frozenset[type[AbstractBuiltinTool]]:
+    """Get the set of all builtin tool types (excluding deprecated tools)."""
+    return frozenset(cls for kind, cls in BUILTIN_TOOL_TYPES.items() if kind not in DEPRECATED_BUILTIN_TOOL_TYPES)
```

Comment:
Instead of having this be a method that always returns the same thing, could this be a constant?